File: app/src/main/java/com/example/intellireview_research_paper/model/reviewmodel.kt
package com.example.intellireview_research_paper.model

data class reviewmodel (
    val paperId:String?=null,
    val userId: String? = null,
    val rating: Number?= 1,
    val comment:String?=null,
    val createdAt: String? = null,
    val updatedAt: String? = null
)

File: BackEnd/model/Reviewmodel.js
const reviewSchema = new mongoose.NumberSchema({
  paperId: {
    type: mongoose.Schema.Types.ObjectId,
    ref: "Paper",
    required: true,
  },
  userId: { type: mongoose.Schema.Types.ObjectId, ref: "User", required: true },
  rating: { type: Number, min: 1, max: 5, required: true },

  comment: { type: String },
  createdAt: { type: Date, default: Date.now },
  updatedAt: { type: Date, default: Date.now },
});

module.exports = reviewSchema;
